### PR TITLE
deno, flutter: add German translation

### PR DESCRIPTION
**Pull request by ecrax**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

File: pages.de/common/deno.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+- Führe eine JavaScript oder TypeScript Datei aus:
+
+`deno run {{pfad/zu/datei.ts}}`
+
+- Starte ein REPL (interaktive Konsole):
+
+`deno`
+
+- Führe eine Datei mit Netzwerkzugriff aus:
```
Comment (kbdharun) [inline pages.de/common/deno.md:14]: I checked this command description with Google translate ( and it translated to `Run a file with network access:`) shouldn't it be `Führen Sie eine Datei mit aktiviertem Netzwerkzugriff aus:` (Run a file with network access enabled:). This is just a small suggestion, feel free to mark this conversation as resolved if this isn't necessary.

```suggestion
- Führen Sie eine Datei mit aktiviertem Netzwerkzugriff aus:
```

**Review by kbdharun — APPROVED** [review]
Welcome to tldr @ecrax, Thanks for your contribution. I have made some suggestions for this page.

File: pages.de/common/deno.md (hunk 1 above)
Comment (ecrax) [reply pages.de/common/deno.md:14]: `Führen Sie` is quite a formal version, whereas the rest of the german translations I found used the imperative version.
`Netzwerkzugriff` vs. `aktiviertem Netzwerkzugriff` is basically `network access` vs. `activated network access`, where I preferred the version in my PR.

Comment (kbdharun) [reply pages.de/common/deno.md:14]: Thanks for the insights 😄 , I think your current version is fine.

**Review by kbdharun — APPROVED** [review]

**Review by blueskyson — APPROVED** [review]

**Pull request merged**